shared/src/map/controls/DefaultTouchHandler.cpp

```diff
@@ -176,7 +189,7 @@ void DefaultTouchHandler::handleTouchUp() {
             LogDebug <<= "TouchHandler: double click detected";
         #endif
         for (auto &listener : listeners) {
```

Comment:
finde iteration wie:
`for (auto const &[index, listener] : listeners) {`
verständlicher bei maps, aber nicht zwingend.